core/Cargo.toml

```diff
@@ -19,7 +19,8 @@ links = "nix"
 bench = false
 
 [features]
-default = ["markdown", "repl", "doc", "format"]
+default = ["markdown", "repl", "doc", "format", "clap"]
```

Comment:
I think it's ok to drop clap from the defaults (we'll just have to remember to bump the crate version). Leaving clap out seems like the right choice absent backwards-compat constraints, and we don't care too much about API stability of the core crates.